### ISSUE TITLE
Update lists and create new ones

### ISSUE THREAD

**Issue by felipenmoura** [felipenmoura/test-your-markup#3]
The lists of words must keep updated.
Also, new lists are quite welcome if you have any in mind :)

**Comment by Eugleo** [comment]
I'd like to work on this issue. I'll add some new lists and will try to update the current ones.

**Comment by felipenmoura** [comment]
Sounds cool @Eugleo :)
I already know of some frameworks missing the "javascript frameworks" list, as expected, that list grows fast!

**Comment by Eugleo** [comment]
Yeah, there's plenty of js frameworks. Do you have any specific in mind?

**Comment by suleka96** [comment]
I added CMD Commands to the list #8

**Comment by suleka96** [comment]
I added 'CMD Commands'  to the list. #8

**Comment by PrismPrince** [comment]
Hi, I add CSS3 properties to the list. 😁 #19
